src/homepageExperience/components/steps/Tokens.tsx

```diff
@@ -62,11 +62,11 @@ export const Tokens: FC<OwnProps> = ({
   )
 
   useEffect(() => {
-    const fetchResources = async () => {
-      await dispatch(getAllResources())
+    const fetchResources = () => {
```

Comment:
As discussed, we removed the `await` here because it wasn't being used to hold up any synchronous code.